specification/components/schemas/responses/generatePatientLetter/422Response.yaml

```diff
@@ -0,0 +1,19 @@
+description: |
+  Where status code 422 (Unprocessable Entity) is returned then an eRS-OperationOutcome-1 will be included in the body, as detailed below. 
+  Check diagnostics property for specific information regarding the error.
+  
+    | Error code                         | Description                                                                                                                  |
+    | ---------------------------------- | ---------------------------------------------------------------------------------------------------------------------------- |
+    | INVALID_REQUEST_TYPE               | The `UBRN` provided exists in e-RS but does not correspond to an active referral.                                            |
+    | NO_ACCESS_CODE                     | The patient does not have an access code/password to log onto the patient portal.                                            |
+    | INVALID_REQUEST_STATE              | The Request/Appointment Request, while otherwise valid, is not in the correct state for the specific action being performed. |
+    | NO_RELATIONSHIP                    | The user does not have a legitimate relationship with the referral.                                                          |
+headers:
```

Comment:
The error code table is not being renderred correctly, the Error code values are under the Description Header and there is no description

Reply:
Yep, this is a know issue and APIM are working to fix it for us (tables in response sections do not render correctly). This is across all endpoints and the issue wasn't there when you were working on your endpoint :)